File: 2339 - Aviões de Papel.cs
using System;
class Script
{
    static private void Main()
    {
        string[] valores = Console.ReadLine().Split(' ');
        int competidores = Convert.ToInt32(valores[0]);
        int papeis = Convert.ToInt32(valores[1]);
        int receber = Convert.ToInt32(valores[2]);
        if (papeis / receber >= competidores) Console.WriteLine("S");
        else Console.WriteLine("N");
    }
}

File: 2339 - Aviões de Papel.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let [competidores, papeis, receber] = lines[0].split(" ")
if(Number(papeis) / Number(receber) >= Number(competidores)) console.log("S")
else console.log("N")
